ckan/config/config_declaration.yaml

```
@@ -1241,16 +1254,29 @@ groups:
 
       - key: ckan.base_templates_folder
         default: templates
-        example: templates
+        example: templates-bs3
         description: |
           This config option is used to configure the base folder for templates used
           by CKAN core. It is currently unused and it only accepts one value: ``templates``
           (Bootstrap 3, the default value from CKAN 2.8 onwards).
 
+      - key: ckan.default.package_type
+        default: dataset
+        description: Default type of dataset that used in UI links(eg. **New Dataset**)
```

```suggestion
        description: Default type of dataset that will be used in the UI links (eg. "New Dataset")
```
Maybe an example would make this clear. Does it mean that if `ckan.default.package_type=record`, the UI will show "New Record"? And the same for groups and orgs

---

Yes, exactly. If you have a `record` package type defined, of course. I and Ian added this functionality 2 years ago. #5337 contains details for the group/organization. The package entity behaves in the same manner. 
This feature potentially provides an easy way to solve issues like #7200 or #5331, where people insist, that the "group" term does not reflect the idea of a group entity. Now they can just create a custom group type and set it as a default one. For non-standard types, that cannot be easily transformed into a human-readable label, chaining [`humanize_entity_type`](https://github.com/ckan/ckan/blob/master/ckan/lib/helpers.py#L1002) helper can solve a problem.